src/components/Actions/GsuiteAuthorize.tsx

```diff
@@ -58,6 +53,11 @@ const Authorize = () => {
     }
   }, [auth_status, history, Jwt.token, setJwt]);
 
+  if (auth_status.is_verified) {
+    let userPath = sessionStorage.getItem("og-path");
+    userPath = userPath === null ? "/" : userPath;
+    return <Redirect to={userPath} />; // use Redirect in place of history.push() to keep from updating during existing state transition
```

Comment:
Not sure I understand the difference between `history.push()` and `<Redirect to={userPath} />`. Could you elaborate a bit more?

Reply:
In the case that a `history.push()` occurs in the middle of a render, it causes React Router to update props to reflect the new path which brings up the `updating during existing state transition` error. By rendering a `<Redirect/>`, we avoid this by redirecting after the rendering has finished.